.github/workflows/test.yml

```diff
@@ -35,3 +35,32 @@ jobs:
         echo "supervisorctl tail -f stellar-core" | docker exec -i stellar sh &
         echo "supervisorctl tail -f horizon" | docker exec -i stellar sh &
         go run test.go
+
+  test-standalone:
+    strategy:
+      matrix:
+        dockerfile: [Dockerfile, Dockerfile.testing]
+        enable_captive_core: [--enable-horizon-captive-core, ""]
```

Comment:
💡 Maybe treat this matrix entry as a list of groups of options that we want tested?
```suggestion
        options: [--enable-horizon-captive-core, ""]
```